### DISCUSSION TITLE
我想在插件里import pymysql

### DISCUSSION THREAD

**Discussion by Qingelish** [nonebot/discussions#395] [通用]
每当我import pymysql时总说我没找到

为什么，nonebot，如何能引用自己做好的其他py文件并传值

**Comment by yanyongyu** [comment]
你安装了pymysql吗

**Reply by Qingelish** [reply]
装了，我自己测试了在其他文件中引用pymsql是可以的，一旦启动机器人他就会报错不知为何

**Comment by Qingelish** [comment]
![1688704267899](https://github.com/nonebot/discussions/assets/19321118/b78cab8e-3aba-402f-8299-99500a81f507)
![Uploading 1688704290227.jpg…]()

**Reply by yanyongyu** [reply]
你的图片没传完

**Reply by Qingelish** [reply]
![mmexport1688706548114](https://github.com/nonebot/discussions/assets/19321118/53f5f417-7531-4ac7-9a9b-e5ad05c1341e)

谢谢.这个部分我不是特别懂，我单位连接很卡

**Reply by yanyongyu** [reply]
你没有安装啊。。。请你在nonebot\myall\.venv这个虚拟环境内安装pymysql
